hw3/01-characters.html

```diff
@@ -6,14 +6,15 @@
     <title>01 - Characters</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <link rel="stylesheet" href="styles.css" />
     <script defer src="01-characters.js"></script>
   </head>
   <body>
     <main>
-      <h1>Exercise 01 - Characters</h1>
-      <section></section>
+      <h1 class="text-center my-5">Exercise 01 - Characters</h1>
+      <section id="list" class="text-center"></section>
```

Comment:
A more descriptive ID could be used if this became a larger codebase, like character-list or list-of-characters, but not really necessary for this exercise.